packages/es-components/src/components/containers/notification/Message.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+interface MessageProps {
+  emphasizedText?: string;
+  text: string;
+}
+
```

Comment:
We should do this like with the other interface.
```suggestion
interface MessageProps extends JSXElementProps<'span'> {
  emphasizedText?: string;
  text: string;
}

```